libs/pages/layout/src/lib/ui/navigation/navigation.tsx

```diff
@@ -40,54 +41,84 @@ export function Navigation({ defaultOrganizationId, clusterNotification }: Navig
         <img className="w-[28px]" src="/assets/logos/logo-icon.svg" alt="Qovery logo" />
       </Link>
```

Comment:
Rename this one `RouterLink` and let our implementation imported as `Link`

Reply:
https://github.com/Qovery/console/pull/1322/commits/9ea6fe2e4c8f1c42ca7802687dd221902a525bd1